Code/Tools/ProjectManager/Source/GemRepo/GemRepoInfo.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QString>
+#include <QDateTime>
+#endif
+
+namespace O3DE::ProjectManager
+{
+    class GemRepoInfo
+    {
+    public:
+        GemRepoInfo() = default;
+        GemRepoInfo(const QString& name, const QString& creator, const QString& summary, const QDateTime& lastUpdated, bool isAdded);
+
+        bool IsValid() const;
+
+        bool operator<(const GemRepoInfo& gemRepoInfo) const;
+
+        QString m_path;
+        QString m_name = "Unknown Gem Repo Name";
+        QString m_creator = "Unknown Creator";
+        bool m_isAdded = false; //! Is the gem currently added and enabled in the project?
```

Comment:
nit: kinda wish we'd use the same name as the UI - "enabled" instead of "added", though in the Gem Catalog it "added" doesn't match the UI which uses "selected".